docker-compose.yml

```diff
@@ -36,7 +36,8 @@ services:
     command: >
       bash -c "wait-for-it -t 30 db:5432 &&
                rm -f tmp/pids/server.pid &&
-               (bundle check || bundle install)
+               (bundle check || bundle install) &&
```

Comment:
I wonder why `bundle check` was in here. I think that `bundle install` is checking that already.

Reply:
It is a habit I got from previous projects, it is a bit more verbose, listing first the missing dependencies and installing them after. We can just keep `bundle install` if you prefer.